client/ayon_core/tools/publisher/control.py

```diff
@@ -655,7 +716,15 @@ def from_data(cls, data):
             PublishPluginActionItem.from_data(action_item)
             for action_item in data["plugin_action_items"]
         ]
-        return cls(error_items, plugin_action_items)
+
+        has_blocking_errors = data["has_blocking_errors"]
```

Comment:
I used direct dictionary access to maintain consistency with the pre-existing code. Since I write my data simultaneously with the pre-existing data, missing keys would indicate an issue with my added code.